code/modules/shuttle/supply.dm

```diff
@@ -77,7 +77,51 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 	if(getDockedId() == "supply_away") // Sell when we get home
 		sell()
 
+/obj/docking_port/mobile/supply/proc/generateManifest(obj/structure/closet/crate/C, var/id, var/ordernum, var/packname) //generates-the-manifests.
```

Comment:
There's already function named like this in supply order, why did you move these, these weren't only used below.

Reply:
Normally, the order's code does the manifest generation. The problem is that for combo boxes, there's no order code being processed when it comes time to generate the combo manifests. Originally, I had duplicated the manifest code to the shuttle's procs, but I was unhappy with having basically the same function in two different spots for two slightly different boxes, so I moved it all over to the shuttle code.

Reply:
You left the other one intact in here though. Remove it if it's unused then.

Reply:
Moved the code back to supply order, details below. Closing both of these comments